### PR TITLE
bug-fix: include currency-greater-than param for 0 value

### LINKED ISSUE #583: Valid GET parameter of 0 filtered out
same problem as described in [py-algorand-sdk](https://app.zenhub.com/workspaces/algorand-main-workspace-5ea71fda6d554d53dda36f16/issues/gh/algorand/py-algorand-sdk/296)

### PR DESCRIPTION
**Pull request by shiqizng**
This PR fixes #583

### CODE REVIEW

File: .test-env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="shiqi/pathupdates"
```
Comment (shiqizng) [inline .test-env:3]: TODO: update branch to V2 before merge.

**Review by algochoi — APPROVED** [review]
Looks good but let's revert to main testing branch before merge

**Review by tzaffi — APPROVED** [review]
lgtm

**Review by winder — CHANGES_REQUESTED** [review]
This is generated code, so there needs to be a corresponding change here: https://github.com/algorand/generator/

**Comment by shiqizng** [comment]
Ran the generator. Included all the updated files.

**Review by winder — APPROVED** [review]

**Review by bbroder-algo — APPROVED** [review]

**Pull request merged**